web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
+    /* on wide screens, this height allows roughly 3 option
+    rows to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    @media (width < $sm_min) {
+        height: 80vh;
```

Comment:
Why not set a viewport-sensitive height like this everywhere, and not just the very tiniest mobile screens? I think it's also probably not the best idea to be letting viewport width determine an element's height.

Reply:
I realized that this particular rule is actually a remnant from a previous iteration of the modal which had more content that would wrap at small widths, increasing its effecting content height.

Now in its current state, the fixed height works just fine at a standard mobile screen, so have removed this rule.